parlai/core/metrics.py

```diff
@@ -23,6 +24,14 @@
     # We'll just turn off things, but we might want to warn the user
     nltkbleu = None
 
+try:
+    import rouge as rouge
+except ImportError:
+    # User doesn't have rouge installed, so we can't use it for rouge
+    # We'll just turn off things, but we might want to warn the user
+    warn_once('Run pip install py-rouge to get rouge metrics')
```

Comment:
Hmm, I think it might be better to NOT do the warning here (after all, if the user imports metrics, but doesn't call _rouge, why are they getting warnings about rouge?)

Then in _rouge, you add the warning.

Also make the warning a bit more:
```
warn_once('Rouge metrics require py-rouge. Please run `pip install py-rouge`.')
```